helios-services/src/main/java/com/spotify/helios/servicescommon/KafkaSender.java

```diff
@@ -39,7 +39,12 @@ public KafkaSender(final Optional<KafkaProducer<String, byte[]>> kafkaProducer)
     this.kafkaProducer = kafkaProducer;
   }
 
-  public void send(final KafkaRecord kafkaRecord) {
+  @Override
+  public boolean isHealthy() {
+    return true;
```

Comment:
since we don't have any issues with the kafka sender and bad behavior when messages can't be published, I opted to leave that out here to avoid changing anything that is working ok today.